web/src/app/core/resolvers/pings.resolver.ts

```diff
@@ -6,22 +6,29 @@ import { ActivatedRouteSnapshot, Resolve, Router } from '@angular/router';
 import { of, Observable } from 'rxjs';
 import { catchError, take } from 'rxjs/operators';
 
-// Third party modules
+// Dashboard hub model and services
```

Comment:
It is written `DashboardHub` - I have tried to fix when I see them

Reply:
okay done